### PR TITLE
Add a function to detect if the system runs a dual PAM stack

### LINKED ISSUE #27: Add a function to detect if the system runs a dual pam stack (64 and 32 bits)
Some YaST modules like yast-auth-client or yast-samba-client adjust the PAM configuration adding or removing PAM modules.

If the system has a dual PAM stack installed (pam-32bit), then these modules will fail if the 32bit version of the PAM module being added or removed isn't installed, e.g. [bug](https://bugzilla.suse.com/show_bug.cgi?id=1190873)

The YaST PAM module should provide a function to check if the system runs a dual PAM stack so each YaST module can add to the list of packages to install the packages providing the 32bit version of the PAM modules when necessary (e.g. samba-winbind-32bit or sssd-32bit).

Just for reference, pam-config implementation of this check: https://github.com/SUSE/pam-config/blob/v1.1/src/sanity_checks.c#L31

### PR DESCRIPTION
**Pull request by scabrero**
Related to bsc#1190873

Fixes #27 

Signed-off-by: Samuel Cabrero <scabrero@suse.de>

### CODE REVIEW

File: src/modules/Pam.rb [hunk 1]
```diff
@@ -135,11 +136,21 @@ def Set(mod, set)
       set ? Add(mod) : Remove(mod)
     end
 
+    # Check if the system runs a dual PAM stack (64 and 32 bits)
+    # @return true if system runs dual PAM stack
+    def DualStack
+      if FileUtils.Exists("/lib/security") && FileUtils.Exists("/lib/libpam.so.0")
```
Comment (ancorgs) [inline src/modules/Pam.rb:142]: Sorry for being nitpicking but I don't see the effective difference between these four lines of code:

```ruby
def somehing
  if condition1 && condition2
    return true
  end
  false
end
```

and a more readable one-liner like:

```ruby
def somehing
  condition1 && condition2
end
```

And looks it's not only me, Rubocop is also complaining about that code.

File: package/yast2-pam.spec [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #
 
 Name:          yast2-pam
-Version:        4.4.0
+Version:        4.4.1
```
Comment (ancorgs) [inline package/yast2-pam.spec:19]: So you are intentionally submitting this only to Tumbleweed. Isn't it?

**Review by ancorgs — CHANGES_REQUESTED** [review]
Please see comments.

By the way, following the conversation in the bug report it looks to me like fixing the problem in YaST is a suboptimal solution.

**Comment by scabrero** [comment]
> Please see comments.
> 
> By the way, following the conversation in the bug report it looks to me like fixing the problem in YaST is a suboptimal solution.

Ok.

**Pull request closed without merging**